skore/src/skore/_sklearn/_estimator/metrics_accessor.py

```diff
@@ -77,12 +77,13 @@ def summarize(
 
         Parameters
         ----------
-        data_source : {"test", "train", "X_y"}, default="test"
+        data_source : {"test", "train", "X_y", "all"}, default="test"
             The data source to use.
 
             - "test" : use the test set provided when creating the report.
             - "train" : use the train set provided when creating the report.
             - "X_y" : use the provided `X` and `y` to compute the metric.
+            - "all" : use both the train and test sets to compute the metrics.
```

Comment:
```suggestion
            - "all" : use both the train and test sets to compute the metrics and present them side by side
```

To make it more explicit that it's not doing an average or anything of that kind